src/Parallelization.php

```diff
@@ -516,4 +517,36 @@ private function runTolerantSingleCommand(
             }
         }
     }
+    
+/**
+     * @param InputInterface $input
```

Comment:
no need for this one, the phpdoc is self-sufficient

Reply:
removed.